corehq/apps/accounting/models.py

```diff
@@ -1049,13 +1053,13 @@ def __str__(self):
 
     def __eq__(self, other):
         return (
-            other is not None
-            and other.__class__.__name__ == self.__class__.__name__
-            and other.plan_version.pk == self.plan_version.pk
-            and other.date_start == self.date_start
-            and other.date_end == self.date_end
-            and other.subscriber.pk == self.subscriber.pk
-            and other.account.pk == self.account.pk
+            other is not None and
+            other.__class__.__name__ == self.__class__.__name__ and
+            other.plan_version.pk == self.plan_version.pk and
+            other.date_start == self.date_start and
+            other.date_end == self.date_end and
+            other.subscriber.pk == self.subscriber.pk and
+            other.account.pk == self.account.pk
```

Comment:
This one I'm not sure of - it seems to be the convention in our codebase to put the line break *before* the binary operator.